packages/theming/default-theme/src/components/Button/ButtonTheme.ts

```diff
@@ -0,0 +1,34 @@
+
+import type { Theme } from '@fluentui-react-native/framework';
+import { immutableMerge } from '@fluentui-react-native/immutable-merge';
+
+import { defaultButtonColorTokens } from './ButtonColorTokens';
+import { defaultButtonFontTokens } from './ButtonFontTokens';
+import { defaultButtonTokens } from './ButtonTokens';
+import { defaultCompoundButtonColorTokens } from './CompoundButtonColorTokens';
+import { defaultCompoundButtonFontTokens } from './CompoundButtonFontTokens';
+import { defaultCompoundButtonTokens } from './CompoundButtonTokens';
+import { defaultToggleButtonColorTokens } from './ToggleButtonColorTokens';
+
+export const defaultButtonTheme = (theme: Theme) =>
+({
+  components: {
+    Button: immutableMerge<object>(
```

Comment:
If you can, you should probably pull the names here from the exported buttonName, etc. from the component

Reply:
default-theme cannot depend on Button, because Button depends on framework which depends on default-theme. :(